invenio_app_ils/ui/src/invenio_app_ils/common/components/ESSelector/ESRelatedSelector.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List, Form, Dropdown, Tab, Label } from 'semantic-ui-react';
+import isEmpty from 'lodash/isEmpty';
+import { ESSelector } from './';
+import { document as documentApi, series as seriesApi } from '../../api/';
+import './ESRelatedSelector.scss';
+import {
+  LanguageRelation,
+  getRelationTypes,
+  getRelationTypeByName,
+  getIconByRelation,
+  getRelationTypeById,
+} from '../RelatedRecords/config';
+
+export default class ESRelatedSelector extends Component {
```

Comment:
Is it important to prefix with `ES`? Me as developer, do I need to know that it is showing results from ES and not DB?
The whole point to use redux is to have the UI **not** to know about the backend.

Reply:
No, I agree. The reason for the naming was that the selector only supports ElasticSearch results but this should probably be refactored to have an ElasticSearch "backend" and then pass the search backend you want to use as a prop to the selector.